templates/lib/prism/serialize.rb.erb

```diff
@@ -147,6 +147,27 @@ module Prism
 
       # variable-length integer using https://en.wikipedia.org/wiki/LEB128
       # This is also what protobuf uses: https://protobuf.dev/programming-guides/encoding/#varints
+      def load_varsint
```

Comment:
Just curious, did you write this code from the Wikipedia page?
I was thinking to use the zig-zag encoding like protobuf but this seems simpler so :+1: